toolkit-distribution/src/release/LICENSE

```diff
@@ -319,10 +319,13 @@ This product bundles the following libraries which are licensed
 under the Apache License v2.
 For details, see their respective project links.
 
+* airline (io.airlift:airline:0.7 - https://github.com/airlift/airline)
```

Comment:
It's already declared on line 401: https://github.com/apache/servicecomb-toolkit/blob/master/toolkit-distribution/src/release/LICENSE#L410

Reply:
Ok, I have removed it in the new commit